src/AssetMapper/SassCssCompiler.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Symfonycasts\SassBundle\AssetMapper;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\Compiler\AssetCompilerInterface;
+use Symfony\Component\AssetMapper\MappedAsset;
+
+class SassCssCompiler implements AssetCompilerInterface
```

Comment:
Can you setup php-cs-fixer - like https://github.com/SymfonyCasts/tailwind-bundle/blob/main/.php-cs-fixer.dist.php - but tweak the library name in the "header"? That'll add the license header to each file.